views/galeria.html

```diff
@@ -0,0 +1,223 @@
+<html lang="es">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>Carpitec 43 | Galería</title>
+		<link rel="icon" href="../img/logoBlanco.svg" type="image/x-icon">
+		<!-- Bootstrap 5  CSS-->
+		<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+		<!-- Material Design Bootstrap 5 -->
+		<link href="https://cdnjs.cloudflare.com/ajax/libs/mdb-ui-kit/3.6.0/mdb.min.css" rel="stylesheet"/>
+		<!-- Fontawesome -->
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+		<!-- Google fonts -->
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium+Web" />
+		<!-- Normalize -->
+		<link rel="stylesheet" href="../css/normalize.css">
+		<!-- Estilos nativos -->
+		<link rel="stylesheet" href="../css/contactanos.css">
+	</head>
+    <body>
+		<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+			<div class="container-fluid">
+				<a class="navbar-brand" href="../index.html">
+					<img src="../img/logoBlanco.svg" alt="logo de Carpitec43" width="60" height="60">
+				</a>
+				<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+				<span class="fas fa-caret-down"></span>
+				</button>
+				<div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+					<div class="navbar-nav">
+						<a class="nav-link hoverlink fw-bold h5 px-1 me-2" href="conocenos.html">Conócenos</a>
+						<a class="nav-link hoverlink fw-bold h5 px-1 me-2" href="contactanos.html">
+							Contáctanos
+						</a>
+                        <a class="nav-link hoverlink fw-bold h5 bg-light text-muted rounded disabled px-1 me-2" href="views/galeria.html">Galería</a>
+					</div>
+				</div>
+			</div>
+		</nav>
+        <header>
+            <div class="container">
+                <div class="ms-3 mt-5">
+                    <h1 class="fw-bold display-4 text-color-main mb-0 text-center">Nuestra Galería</h1>
+                    <h3 class="fw-bold ms-2 text-center">diseños que seguro te gustaran</h3>
+                </div>
+            </div>
+        </header>
+        <main class="bg-color-light position-relative">
+            <div class="container pt-5 pb-lg-3 pb-1">
+                <div class="row pb-lg-4 pb-2">
+					<div class="col-lg-6 col-12 pe-lg-4 pe-3 ps-l-0 ps-3 pb-3 pb-0 position-relative">
+						<img
+							class="galery-img-1 border-gelery-img object-fit-cover opacidad-negro"
+							src="../img/galeria/galeria-1.jpeg" alt="galeria-1"
+						>
```

Comment:
Necesitamos la propiedad loadin="lazy" en todas las imagenes para aligerar la carga según el scroll o qué opinas?
```suggestion
						<img
						        loading="lazy"
							class="galery-img-1 border-gelery-img object-fit-cover opacidad-negro"
							src="../img/galeria/galeria-1.jpeg" alt="galeria-1"
						>
```